app/views/admin/categories/index.html.erb

```diff
@@ -0,0 +1,39 @@
+<div class="card">
+    <div class="card-header">       
+        <h3> Admin | Categories 
+        <%= link_to "Add Category", new_admin_category_path, class: "btn btn-outline-info float-right" %>
+        </h3>
+    </div>
+        <table class="table table-hover">
+            <thead>
+                <tr>
+                <th scope="col">Title</th>
+                <th scope="col">Description</th>
+                <th scope="col">Words</th>
+                <th scope="col"></th>
+                <th scope="col"></th>
+                </tr>
+            </thead>
+
+            <tbody>
+                <% @categories.each do |category| %>
+                    <tr>
+                    <td><%= link_to category.title, admin_category_path(category), style: "color:#007acc" %></td>
+                    <td><%= category.description %></td>
+                    <td></td>
+                    <td>
+                        <%= link_to edit_admin_category_path(category), style: "color:gray;" do %>
```

Comment:
Please use the bootstrap class, 'text-muted' and remove 'style: color: gray',
The text-muted class makes the icon color gray...